src/renderer/entities/chain/ui/ChainTitle/ChainTitle.tsx

```diff
@@ -32,9 +32,7 @@ export const ChainTitle = ({ as: Tag = 'div', fontClass, className, iconSize = 1
   return (
     <Tag className={cnTw('flex items-center gap-x-2', className)}>
       <ChainIcon src={chainObj?.icon} name={chainObj?.name} size={iconSize} />
-      <TextBase as="span" className={cnTw('text-text-tertiary text-footnote', fontClass)}>
```

Comment:
I assume `<TextBase />` should be used as a template component only inside `Typography`